### ISSUE TITLE
Resource details layout - assets

### ISSUE THREAD

**Issue by SeanyB** [Shopify/polaris-react-archive#7934]

**Comment by johanstromqvist** [comment]
- [x] How it helps merchants
- [x] Useful to know
  - [x] Use default width
  - [x] Group similar content
  - [x] Primary column
  - [x] Secondary column
  - [x] Arrange content

![Image](https://user-images.githubusercontent.com/38329124/212435587-983cee01-9822-415f-919c-4c341c276b29.png)

**Comment by johanstromqvist** [comment]
I _think_ we should exclude the `save button` from the footer. The preferred pattern is the save bar appearing when an edit is made. It seems excessive and sets the wrong expectations to keep it.

Thoughts @SeanyB @raquelbreternitz 

![Image](https://user-images.githubusercontent.com/38329124/212435770-2490172c-d1b2-4b40-b2ec-d53ad4cdea61.png)

**Comment by SeanyB** [comment]
> I think we should exclude the save button from the footer. The preferred pattern is the save bar appearing when an edit is made. It seems excessive and sets the wrong expectations to keep it.

I agree, let's **not** include the Save button at the bottom of the layout.

**Comment by johanstromqvist** [comment]
At a closer look, this is super inconsistent in our core resources.

It makes sense that pages would vary a bit depending on its purpose, but page actions such as `duplicate`, `save`, `archive` and `delete` must always be easy and intuitive to find. Currently it's all over the place, and this is a pattern docs' job to fix.

Suggested guidelines
- No footer actions
- No save action
- Default to "More actions" for page actions
- Surface single page actions if extremely common.

# Audit
## Products
Products has no page actions in "More actions", but `duplicate` in common actions, and `archive`, `delete` and `save` in footer actions.
<img width="1242" alt="image" src="https://user-images.githubusercontent.com/38329124/212752547-1227c7f9-2408-4b47-84e3-876ddf594a92.png">
<img width="1060" alt="image" src="https://user-images.githubusercontent.com/38329124/212753900-2d9a88d1-25c3-45ca-b334-a4934a3c653b.png">

## Orders
Orders has `cancel order` and `archive` in "More actions", and no footer actions
<img width="1206" alt="image" src="https://user-images.githubusercontent.com/38329124/212752289-6700c3ed-f1c5-4ea9-8987-ab34ff174012.png">
<img width="1191" alt="image" src="https://user-images.githubusercontent.com/38329124/212754023-c8cf1a9f-3955-4f2a-b315-b5c39060c9c4.png">

## Customers
Customers has TWO `delete` actions, one in "More actions" and one in the footer. Plus the destructive button has wrong styling.

_[ Screenshot removed ]_

<img width="857" alt="image" src="https://user-images.githubusercontent.com/38329124/212751933-e7bf5aeb-5a8c-436d-b21e-bd2b53cbc2e6.png">

## Discounts
Discounts has no "More actions", but `duplicate` among common actions and `delete` in footer actions
<img width="1178" alt="image" src="https://user-images.githubusercontent.com/38329124/212751731-06b318f3-daab-4315-a363-63b3840111ff.png">

<img width="1214" alt="image" src="https://user-images.githubusercontent.com/38329124/212751640-d3d85776-6351-452e-aa79-a300225f79ec.png">

**Comment by johanstromqvist** [comment]
^ this is not an "assets" problem, so I'll create a separate issue for this inconsistency problem.

**Comment by raquelbreternitz** [comment]
Considering the ongoing conversation in #8058, should we exclude footer actions altogether?

I'm beginning to wonder if "Page actions" overall, including the top bar and the footer, should be their own pattern, which is then associated with this one.

Or should we wait on this as a dependency on what we decide in #8058?

**Comment by raquelbreternitz** [comment]
All assets updated and in place [on Figma](https://www.figma.com/file/JUKo15oKz1aCcdeNP4UxyE/Reboot-Polaris-Patterns?node-id=4216%3A181657&t=HA0CM6GnvLwebb3p-1) 

Styling FYI: Made the executive decision to use no blur; instead, highlighting/subduing of elements will be done by "low-fi-ing" elements, i.e. lightening icons and erasing or replacing text with squirqles like so:

![Image](https://user-images.githubusercontent.com/13204374/213813330-6c735cad-ec95-4410-aa72-28b69a8ec42d.png)

I updated the "Visual Assets" page to reflect this, but I want to pull all of those elements out and create asset templates for future teams. Made myself an issue for that [here](https://github.com/Shopify/polaris/issues/8122).

**Comment by raquelbreternitz** [comment]
All assets clean and done, ready for fresh eyes 👍

**Comment by johanstromqvist** [comment]
Thanks @raquelbreternitz !

> Considering the ongoing conversation in https://github.com/Shopify/polaris/issues/8058, should we exclude footer actions altogether?

I think we resolved this elsewhere, but yes, let's omit the footer actions and then _probably_ include a "useful to know" about order of secondary actions.

> I'm beginning to wonder if "Page actions" overall, including the top bar and the footer, should be their own pattern, which is then associated with this one.

I think the page layout patterns should give clarity in "where to put common things", so I'd like to include secondary actions in that. But perhaps there's a another tangental pattern here? What are you picturing? 

> Or should we wait on this as a dependency on what we decide in https://github.com/Shopify/polaris/issues/8058?

Let's move forward with what we got. It will be an easy addition if we e.g. create a new "useful to know" later.

**Comment by johanstromqvist** [comment]
@raquelbreternitz are these all done?

If so, could you move the issue to `this week`, remove `design` and add `engineering`, and unassign yourself?

Oooor, move it to `in progress`, keep yourself assigned, and give that code base a poke? 🙃

**Comment by raquelbreternitz** [comment]
@johanstromqvist gonna attempt to poke the codebase, if I fail I'll assign it over. wish me luck!

**Comment by raquelbreternitz** [comment]
Addressed this with #8160 so closing this now
